tools/bump-version.py

```
@@ -101,7 +101,9 @@ def cli(project: str | Path, version: str) -> None:
             VERSION_PATCH=patch,
         )
         # also bump language JSONs
-        subprocess.run(["python", project / "translations" / "cli.py", "gen"])
+        subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)